static/src/stylesheets/module/journalism/_campaigns.scss

```diff
@@ -1,25 +1,204 @@
 
 .campaign--snippet {
     border-top: 1px $garnett-neutral-4 solid;
-    background: $garnett-neutral-8;
+    border-bottom: 1px $garnett-neutral-4 solid;
     position: relative;
-    padding: 0 0 6px;
-    margin: 16px 0 36px;
-    font-size: 14px;
-    line-height: 21px;
+    padding: 0;
+    margin: 16px -10px 36px;
+    @include mq(phablet) {
+        margin: 16px 0 36px -10px;
+    }
 
     &:not([open]) .is-on,
     &[open] .is-off {
         display: none;
     }
+
+    &[open] {
+        background: $garnett-neutral-8;
+    }
+
+    p {
+        font-size: 14px;
+        line-height: 18px;
+        font-family: $f-serif-headline;
+        font-weight: 400;
+        padding-top: 2px;
+    }
+}
+
+article {
+    .speech-bubble {
+        color: $garnett-neutral-1;
+        background-color: $news-garnett-highlight;
+        &::after {
+            color: $garnett-neutral-1;
+            background-color: $news-garnett-highlight;
+        }
+    }
+    .form_submit {
```

Comment:
I'd be more comfortable if this was limited to a callout, e.g. `.campaign--snippet .form_submit button`